template

```diff
@@ -12,7 +12,7 @@
  * permissions and limitations under the License.
  */
 
-package lints
+package PACKAGE
 
 import (
 	"github.com/zmap/zcrypto/x509"
```

Comment:
Would you mind also adding the missing `github.com/zmap/zlint/lint` import to the template?